backend/capellacollab/users/crud.py

```diff
@@ -48,6 +48,16 @@ def update_role_of_user(
     return user
 
 
+def update_last_login(
+    db: Session, user: DatabaseUser, last_login: datetime | None = None
+) -> DatabaseUser:
+    if not last_login:
+        last_login = datetime.now()
+    user.last_login = last_login
```

Comment:
This doesn't work when a new user signs in (the user is not created and therefore None).

```
AttributeError: 'NoneType' object has no attribute 'last_login'
```